Recipely/Recipely/Services/NetworkService/DTO/DishTDO.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  DishTDO.swift
+//  Recipely
+//
+//  Created by Tixon Markin on 13.03.2024.
+//
+
+import Foundation
+
+/// Структура, представляющая информацию о конкретном блюде.
+struct DishDTO: Codable {
+    /// URI блюда.
+    let uri: String
+    /// Название блюда.
+    let label: String
+    /// URL-адрес изображения блюда.
+    let image: String
+    /// Общее время приготовления блюда в секундах.
```

Comment:
```suggestion
    /// Общее время приготовления блюда в секундах.
```
C доки берем все коменты